### ISSUE TITLE
Rule Proposal: max selector chain length

### ISSUE THREAD

**Issue by kajyr** [CSSLint/csslint#539]
It's useful, specially when working with less or sass and partials, to have a check on the selector length. It would be nice a rule that checks if selectore are too much nested (by looking at the compiled css selector length).

Example:

``` css
#myApp .main .column #subSection .toolbar .left ul li a .icon
```

should trigger a warning.
